src/code/audio/code_8019AF00.c

```diff
@@ -217,768 +236,893 @@ s8 gUnderwaterSfxReverbAdd = 0;
+    0,                                    // NA_BGM_FROG_SONG
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_SONATA
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_LULLABY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_NEW_WAVE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_ELEGY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_OATH
+    0,                                    // NA_BGM_MAJORAS_LAIR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_LULLABY_INTRO
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_GUITAR_BASS_SESSION
+    SEQ_FLAG_FANFARE,                     // NA_BGM_PIANO_SESSION
+    SEQ_FLAG_FANFARE,                     // NA_BGM_INDIGO_GO_SESSION
+    SEQ_FLAG_ENEMY,                       // NA_BGM_SNOWHEAD_TEMPLE
+    SEQ_FLAG_ENEMY,                       // NA_BGM_GREAT_BAY_TEMPLE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_NEW_WAVE_SAXOPHONE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_NEW_WAVE_VOCAL
+    0,                                    // NA_BGM_MAJORAS_WRATH
+    0,                                    // NA_BGM_MAJORAS_INCARNATION
+    0,                                    // NA_BGM_MAJORAS_MASK
+    SEQ_FLAG_FANFARE,                     // NA_BGM_BASS_PLAY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_DRUMS_PLAY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_PIANO_PLAY
+    SEQ_FLAG_ENEMY,                       // NA_BGM_IKANA_CASTLE
+    0,                                    // NA_BGM_GATHERING_GIANTS
+    SEQ_FLAG_FANFARE_KAMARO,              // NA_BGM_KAMARO_DANCE
+    0,                                    // NA_BGM_CREMIA_CARRIAGE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_KEATON_QUIZ
+    0,                                    // NA_BGM_END_CREDITS_1
+    0,                                    // NA_BGM_OPENING_LOOP
+    0,                                    // NA_BGM_TITLE_THEME
+    SEQ_FLAG_FANFARE,                     // NA_BGM_DUNGEON_APPEAR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_WOODFALL_CLEAR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_SNOWHEAD_CLEAR
+    0,                                    //
+    0,                                    // NA_BGM_INTO_THE_MOON
+    0,                                    // NA_BGM_GOODBYE_GIANT
+    0,                                    // NA_BGM_TATL_AND_TAEL
+    0,                                    // NA_BGM_MOONS_DESTRUCTION
+    0,                                    // NA_BGM_END_CREDITS_2
 };
 
 s8 sSpecReverbs[20] = {
     0, 0, 0, 0, 0, 0, 0, 40, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 };
 
-NatureAmbienceDataIO sNatureAmbienceData[20] = {
-    // natureSeqId: 0
+typedef struct {
+    /* 0x0 */ u16 initChannelMask;     // bitpacked channel mask to initialize channels
+    /* 0x2 */ u16 initMuteChannelMask; // bitpacked channel mask to mute channel upon initialization
+    /* 0x4 */ u8 channelProperties[3 * 33 + 1];
```

Comment:
This is a strange number, where does it come from?

Reply:
Each ambience entry has three numbers associated with it (e.g. `channel, ioPort, ioData`), so that's where the 3 comes from. Then there are 33 entries, so 3 * 33. Then, the termination command is only 1 entry `0xFF`. So that's where the +1 comes from

Reply:
Ah, so will the 33 be getting a define, at least?

Reply:
Still waiting for an answer here.

Reply:
Yeah, I've gone and made a define for it